packages/ceramic-cli/package.json

```diff
@@ -45,8 +45,8 @@
     "commander": "^6.0.0",
     "express": "^4.17.1",
     "identity-wallet": "2.0.0-alpha.1",
-    "ipfs": "^0.49.0",
-    "ipfs-http-client": "^46.0.0"
+    "ipfs": "github:simonovic86/ipfs",
+    "ipfs-http-client": "github:simonovic86/ipfs-http-client"
```

Comment:
What's going on here?

Reply:
The `createDagJWS` method that you provided in `js-did` library uses, amongst other things, CID v1 and `ipld-dag-cbor` v0.17.0 which is not compatible with one used in the latest IPFS release. That's why I forked the js-ipfs and tested it with the latest release. Because of this incompatibility, we're currently blocked with the integration.